### PR TITLE
Update 'About Us' information for Siddarth

### PR DESCRIPTION
**Pull request by siddarth2824**

### CODE REVIEW

File: src/main/main.iml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="true">
+    <exclude-output />
+    <content url="file://$MODULE_DIR$">
+      <sourceFolder url="file://$MODULE_DIR$/java" isTestSource="false" />
+    </content>
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+</module>
```
Comment (ianyong) [inline src/main/main.iml:11]: Don't commit `.iml` files.

File: docs/team/siddarth2824.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+* **Enhancements to existing features**:
+
+* **Documentation**:
+  * User Guide:
+  * Developer Guide:
+
+* **Community**:
+
+* **Tools**:
```
Comment (ianyong) [inline docs/team/siddarth2824.md:30]:
```suggestion
* **Tools**:

```

Newline at EOF.

**Review by ianyong — DISMISSED** [review]
LGTM!

File: docs/AboutUs.md [hunk 3]
```diff
@@ -29,14 +29,14 @@ You can reach us at the email <ay2021s1-cs2103t-w16-3@googlegroups.com>
...
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/siddarth2824.png" width="200px">
 
-[[github](http://github.com/johndoe)] [[portfolio](team/johndoe.md)]
+[[github](http://github.com/siddarth2824)] [[portfolio](team/siddarth2824.md)]
 
 * Role: Developer
-* Responsibilities: Data
+* Responsibilities: Scheduling and Tracking, User Interface-In-Charge
```
Comment (yongping827) [inline docs/AboutUs.md:39]:
```suggestion
* Responsibilities: Scheduling and Tracking, User Interface
```

**Review by yongping827 — CHANGES_REQUESTED** [review]

**Review by yongping827 — APPROVED** [review]
LGTM

**Review by ianyong — APPROVED** [review]
LGTM!

**Pull request merged**